aea/connections/local/connection.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, loop: AbstractEventLoop = None):
         self._loop = loop if loop is not None else asyncio.new_event_loop()
         self._thread = Thread(target=self._run_loop)
 
-        self._in_queue = asyncio.Queue(loop=self._loop)  # type: asyncio.Queue
+        self._in_queue = None  # type: Optional[asyncio.Queue]
```

Comment:
This change is due to the deprecation of `loop` parameter of `asyncio.Queue` from Python 3.8:
https://docs.python.org/3/library/asyncio-queue.html#asyncio.Queue

that implies the asyncio queue must be initialized when the right loop has been set - in this context, in `_run_loop`

Reply:
Ok. How did this not cause issues earlier?

Reply:
it was just a warning when using Python 3.8 (which has been ignored), it does not affect functionalities. But it will when the `loop` parameter will be removed in Python 3.10.